src/domain/RootViewModel.js

```diff
@@ -65,7 +68,13 @@ export class RootViewModel extends ViewModel {
                     this._showSessionLoader(sessionId);
                 }
             }
-        } else {
+        } else if (ssoSegment) {
+            this.urlCreator.normalizeUrl();
+            if (this.activeSection !== "login") {
+                this._showLogin({loginToken: ssoSegment.value});
```

Comment:
I think we discussed making this just the loginToken rather than an object before. Any reason to keep it this way?